wazo_call_logd/cel_interpretor.py

```diff
@@ -252,6 +254,38 @@ def interpret_wazo_meeting_name(self, cel, call):
 
         return call
 
+    def interpret_wazo_user_missed_call(self, cel, call):
+        extra = extract_cel_extra(cel.extra)
+        if not extra:
+            return
+
+        extra_tokens = extra['extra'].split(',')
+        wazo_tenant_uuid = extra_tokens[0].split(': ')[1]
+        source_user_uuid = extra_tokens[1].split(': ')[1]
+        destination_user_uuid = extra_tokens[2].split(': ')[1]
+        destination_exten = extra_tokens[3].split(': ')[1]
+        source_name = extra_tokens[4].split(': ')[1]
+        destination_name = extra_tokens[5].split(': ')[1]
+
+        source_participant = CallLogParticipant(
```

Comment:
What happens on an incoming call? My guess is that `source_user_uuid` is going to be `""` I'm not sure.
You should add another test that covers this scenario.

Reply:
To be consistent with other usage of `CallLogParticipant` the `line_id` should be added if possible. The channel variable `WAZO_LINE_ID` holds the ID of the line of the caller, if a user called. It could be added to the `CELGenUserEvent` to be added here.

Reply:
I have added the `line_id` to both participants as per your request. Can you please elaborate more what you mean in your first comment about the `source_user_uuid` and the incoming call? does it mean that I have to include the same interpreter and from the `CalleeCELInterpretor` side? Thanks